src/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[project]
+name = "NNPOps"
+description = "Highly optimized, open source implementations of bottleneck operations in NNPs"
+authors = [{ name = "OpenMM", email = "info@openmm.org" }]
```

Comment:
I don't think that address exists?

Reply:
Yes you are right, it was just a placeholder since I didn't know what to put. Feel free to suggest an email

Reply:
Let's not list an email.  The way for people to contact us is through the github repo.

Reply:
Done